src/types/reducers.js

```diff
@@ -136,12 +136,12 @@ export type ZippedProfilesState = {
 };
 
 export type UrlState = {|
-  dataSource: DataSource,
-  hash: string,
-  profileUrl: string,
-  selectedTab: TabSlug,
-  pathInZipFile: string | null,
-  profileSpecific: {|
+  +dataSource: DataSource,
+  +hash: string,
+  +profileUrl: string,
+  +selectedTab: TabSlug,
+  +pathInZipFile: string | null,
+  +profileSpecific: {|
     selectedThread: ThreadIndex | null,
```

Comment:
Here the problem wasn't with the definition of `Reducer`, but with the definition for `combineReducers`... but otherwise similar than the one above.